jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -98,29 +98,38 @@ pipeline {
                         currentBuild.result = 'ABORTED'
                         error("OSD Version $version does not match OS Version $versionOpenSearch")
                     }
-
                 }
             }
-        }
-        stage('integ-test') {
-            agent {
-                docker {
-                    label AGENT_LABEL
-                    image docker_images["$distribution"]
-                    args docker_args["$distribution"]
-                    registryUrl 'https://public.ecr.aws/'
-                    alwaysPull true
+            post {
+                always {
+                    postCleanup()
                 }
             }
+        }
+        stage('integ-test') {
+            // Required running on agent directly here to trigger docker stages in agent node, not trigger docker within docker container
```

Comment:
Did not understand the wording here for "not trigger docker within docker container"

Reply:
That is copied from the OS part, where we have to make sure the run is happened directly on an agent not a docker container, so that the agent can trigger more docker container while it is within an agent, this cannot happen if you start in a docker container.

Reply:
Add the same comment change on OS as well. Thanks.